09Ianuarie/src/ro/sci/bv/FahrenheitToCelsiusConversion/FahrenheitToCelsiusConversion.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * This program prompts the user to insert a temperature in Fahrenheit degrees
+ * Then it realizes a conversion to Celsius degrees and outputs a message, based on the inputted value
+ *
+ * @author Bogdan Butuza
+ */
+package ro.sci.bv.FahrenheitToCelsiusConversion;
+import java.text.DecimalFormat;
+import java.util.Scanner;
+public class FahrenheitToCelsiusConversion {
+
+    /*To display the double variable with only one decimal */
+    private static DecimalFormat oneDecimal = new DecimalFormat(".#");
+
+    /**
+     * This method converts a temperature from Fahrenheit degrees to Celsius degrees
+     * If the measured temperature in Celsius is higher than 37 degrees, the method warns the users that
+     * they are ill
+     * If the inputted temperature is right, an message with its conversion to Fahrenheit will be printed to
+     * the console
+     * @param temperature a double value, representing the temperature to be converted
+     */
+    private static void toCelsius(double temperature) {
```

Comment:
this method does too much. You should have a method that just does the conversion and returns the value as a double.
The rest of the check and other System.out.println would be better suited in another method or in main funciton.